src/elements/content-sidebar/versions/messages.js

```diff
@@ -3,6 +3,11 @@
 import { defineMessages } from 'react-intl';
 
 const messages = defineMessages({
+    fileRequestDisplayName: {
+        defaultMessage: 'File Request',
+        description: 'File Request feature display name',
```

Comment:
We may want to provide a bit more context so the translator can provide a meaningful translation.